defaults/main.yml

```diff
@@ -155,8 +155,8 @@ rke2_disable_cloud_controller: false
 # applicable only if rke2_disable_cloud_controller is true
 rke2_cloud_provider_name: "external"
 
-# Path to custom manifests deployed during the RKE2 installation
-# It is possible to use Jinja2 templating in the manifests
+# Path to folder with custom manifests deployed during the RKE2 installation
+# It is possible to use Jinja2 templating in the manifests, if the filename ends with .j2
```

Comment:
It can be also yaml or yml.

`ansible.builtin.template` will template file with any file extension. The `.j2` is not mandatory. You may use just .`yaml` and the file will be still templated if there is a Jinja2 syntax in it.

This option was meant to be used for files with yaml (or yml) files. But user could use Jinja2 syntax in it and it would be templated.

The actual bug here is that if the user will add `.j2` file extension to the manifest file, the file will end up with `.j2` also on the RKE2 server.